src/pywink/devices/sensors.py

```diff
@@ -93,7 +93,13 @@ def humidity_percentage(self):
         :return: The relative humidity detected by the sensor (0% to 100%)
         :rtype: int
         """
-        return self.last_reading()
+        # The subscription response returns a deciaml not an int
```

Comment:
See: https://github.com/bradsk88/python-wink/pull/30/files#r62879581

Remove these changes and add an implementation of `pubnub_update` to this class that adapts the pubnub data to match the format of the Wink Data.